src/requests/flow.ts

```diff
@@ -264,3 +268,44 @@ export async function _destroyPublishedFlow(
     );
   return Boolean(response.delete_published_flows_by_pk?.id);
 }
+
+interface SetFlowStatus {
+  flow: {
+    id: string;
+    status: FlowStatus;
+  };
+}
+
+async function setStatus(
+  client: GraphQLClient,
+  args: { flow: { id: string }; status: FlowStatus },
+) {
+  try {
+    const { flow } = await client.request<SetFlowStatus>(
+      gql`
+        mutation SetFlowStatus(
+          $flowId: uuid!
+          $status: flow_status_enum_enum!
```

Comment:
`_enum_enum` looks like a possible typo?

Reply:
This is actually correct! The table has the suffix `_enum` and then the type Hasura generates adds it on again annoyingly. This happens a few other places, e.g. https://github.com/theopensystemslab/planx-new/blob/main/api.planx.uk/modules/send/utils/helpers.ts#L86

I think the solution is not to name enum tables `*_enum`?